src/scripts/events/domEvents.js

```diff
@@ -56,10 +57,10 @@ const domEvents = () => {
     if (e.target.id.includes('add-author-btn')) {
       addAuthorForm();
     }
-    // FIXME: ADD CLICK EVENT FOR EDITING AN AUTHOR
-    if (e.target.id.includes('edit-author-btn')) {
+    // DONE: ADD CLICK EVENT FOR EDITING AN AUTHOR
+    if (e.target.id.includes('update-author')) {
       const [, authorFirebaseKey] = e.target.id.split('--');
-      getSingleAuthor(authorFirebaseKey).then((authorObject) => addAuthorForm(authorObject.uid, authorObject));
+      getSingleAuthor(authorFirebaseKey).then((authorObject) => addAuthorForm(authorObject));
```

Comment:
https://github.com/lesleyjkeyes/IN-CLASS-Almost-Amazon/pull/6/files#r890623316

Same thing here!